plugin-flex-ts-template-v2/src/feature-library/dispositions/custom-components/DispositionTab/DispositionTab.tsx

```diff
@@ -0,0 +1,143 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Actions, ITask, Manager, Template, templates, withTaskContext } from '@twilio/flex-ui';
+import { Radio, RadioGroup } from '@twilio-paste/core/radio-group';
+import { Stack } from '@twilio-paste/core/stack';
+import { TextArea } from '@twilio-paste/core/textarea';
+import { Label } from '@twilio-paste/core/label';
+import { HelpText } from '@twilio-paste/core/help-text';
+import { Box } from '@twilio-paste/core/box';
+import { debounce } from 'lodash';
+
+import { getDispositionsForQueue, isNotesEnabled, isRequireDispositionEnabled } from '../../config';
+import AppState from '../../../../types/manager/AppState';
+import { reduxNamespace } from '../../../../utils/state';
+import { updateDisposition, DispositionsState } from '../../flex-hooks/states';
+import { StringTemplates } from '../../flex-hooks/strings';
+
+export interface OwnProps {
+  task?: ITask;
+}
+
+const DispositionTab = (props: OwnProps) => {
+  const NOTES_MAX_LENGTH = 100;
```

Comment:
is 100 arbitrary or is it aligned with the max length of the field in flex insights?

Reply:
100 "may be" (I don't really know why the language uses that term in the docs!) the max length of the Content field per the Insights docs: https://www.twilio.com/docs/flex/end-user-guide/insights/data-model#attributes-1